### PR TITLE
Create build.yml

### PR DESCRIPTION
**Pull request by stickyPiston**

### CODE REVIEW

**Comment by stickyPiston** [comment]
This is a very rudimentary workflow that runs the linter, builds the application and saves the dist folder as an artefact. Before merging I still want to check whether there are some specifications sad-server requires for the artefact so that it can conveniently use it.

File: index.html [hunk 1]
```diff
@@ -11,7 +11,6 @@
 <body>
   <div id="root"></div>
   <script type="module" src="/src/main.jsx"></script>
-  <script src="snow.js"></script>
```
Comment (SilasPeters) [inline index.html:14]: Y no snow? :(

File: src/components/Activity.jsx [hunk 2]
```diff
@@ -8,6 +7,8 @@ export default class Activity extends Component {
     super(props);
   }
 
+  dom_node = createRef(null);
+
```
Comment (SilasPeters) [inline src/components/Activity.jsx:11]: Could you add a comment on why this is needed?

**Review by SilasPeters — CHANGES_REQUESTED** [review]
Great addition! Few nitpicks here. Also, could you merge main into this PR first?

**Comment by SilasPeters** [comment]
> Before merging I still want to check whether there are some specifications sad-server requires for the artefact so that it can conveniently use it.

Any updates on this?

**Comment by SilasPeters** [comment]
@stickyPiston just reminding you that this PR has been reviewed

**Comment by SilasPeters** [comment]
This might close #64

**Review by SilasPeters — CHANGES_REQUESTED** [review]
Merge master, and I would approve this!

**Pull request closed without merging**